slash/utils/traceback_utils.py

```diff
@@ -258,10 +259,14 @@ def distill_object_attributes(obj, truncate=True):
 
 
 def _safe_repr(value, truncate=True):
-    try:
-        returned = repr(value)
-    except Exception:  # pylint: disable=broad-except
-        return "[Unprintable {!r} object]".format(type(value).__name__)
+    ignored_classes = config.root.log.traceback_ignored_classes
+    if ignored_classes and isinstance(value, tuple(ignored_classes)):
```

Comment:
Build the tuple once at the top of the stack and pass it around, otherwise it would be too costly